tests/test_autonomy/test_configurations.py

```diff
@@ -139,3 +144,27 @@ def teardown(
 
         os.chdir(cls.cwd)
         shutil.rmtree(cls.t)
+
+
+@pytest.mark.parametrize(
+    "service_file",
+    Path(
+        ROOT_DIR,
+        "tests",
+        "data",
+        "dummy_service_config_files",
+    ).iterdir(),
+)
+def test_load_service(service_file: Path) -> None:
+    """Test loading and processing service component."""
+
+    with open_file(service_file, "r", encoding="utf-8") as fp:
+        data = yaml_load_all(fp)
+
+    data = cast(List[Dict], apply_env_variables(data, env_variables=os.environ))
+    service_config, *overrides = data
+    Service.validate_config_data(service_config)
+    service_config["license_"] = service_config.pop("license")
```

Comment:
this is somehow needed because it's a reserved variable name? :confused:

Reply:
Yes